third_party/blink/renderer/includes.gni

```diff
@@ -1,29 +1,25 @@
 # common includes which can help minimize patches for
```

Comment:
you can probably just drop these changes? Also fine if you leave them I guess, but you don't have to fix the lint errors if you don't touch it :)